lib/api/response/order_response.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'order_response.g.dart';
+
+@JsonSerializable()
+class OrderResponse {
+  final String id;
+  @JsonKey(name: 'created_at')
+  final String createdAt;
+  @JsonKey(name: 'customer_name')
+  final String customerName;
+  final double price;
```

Comment:
Always handle the incoming data from third party as nullable for safety

```suggestion
  final String? id;
  @JsonKey(name: 'created_at')
  final String? createdAt;
  @JsonKey(name: 'customer_name')
  final String? customerName;
  final double? price;
```